shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/model/Dataset.java

```diff
@@ -75,7 +75,7 @@ public abstract class Dataset extends AbstractEntity {
 
 	/** Dataset expression list. */
 	@JsonIgnore
-	@OneToMany(fetch = FetchType.LAZY, mappedBy = "dataset", cascade = CascadeType.ALL)
+	@OneToMany(fetch = FetchType.EAGER, mappedBy = "dataset", cascade = CascadeType.ALL)
```

Comment:
Dear @julien-louis, I am really wondering, why this change here is to "eager" and same below for the dataset_files. The performance impact is really, really bad here, when each time, we get a dataset, we grab expression + files with eager, what results in huge query results.

Reply:
done, seems to work with lazy